invenio_rdm_records/fixtures/demo.py

```diff
@@ -158,7 +155,7 @@ def create_fake_record():
             }],
             "publisher": "InvenioRDM",
             "publication_date": fake_edtf_level_0(),
-            "subjects": CachedVocabularies.fake_subjects(),
+            "subjects": [],  # Too complex to generate for now
```

Comment:
Yes, I will mention it at standup again. The demo instance could also have an `app_data/` with some small number of subjects so that when people trying out the online demo can add a subject to a record.